subsys/mgmt/ec_host_cmd/backends/ec_host_cmd_backend_uart.c

```diff
@@ -0,0 +1,303 @@
+/*
+ * Copyright (c) 2022 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <zephyr/drivers/uart.h>
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/mgmt/ec_host_cmd/backend.h>
+#include <zephyr/mgmt/ec_host_cmd/ec_host_cmd.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/sys/time_units.h>
+#include <zephyr/sys/util.h>
+
+LOG_MODULE_REGISTER(host_cmd_uart, CONFIG_EC_HC_LOG_LEVEL);
+
+/*
+ * Enumeration to maintain different states of incoming request from
+ * host
+ */
+enum uart_host_command_state {
+	/*
+	 * USART host command handler not enabled.
+	 */
+	USART_HOST_CMD_STATE_DISABLED,
+
+	/*
+	 * Ready to receive next request
```

Comment:
nit: either terminate the "titles" with a `.` or don't. Or drop these "titles" altogether and just keep the description, it's not really adding much to the enum name anyway.

Reply:
Done.